### PR TITLE
Addresses #2791 relationship update

### PR DESCRIPTION
**Pull request by aleixpuigb**
Addresses #2791 relationship update

### CODE REVIEW

File: src/ontology/uberon-edit.obo [hunk 1]
```diff
@@ -44033,7 +44033,7 @@ xref: UMLS:C0006270 {source="ncithesaurus:Bronchiole"}
 xref: VHOG:0000675
 xref: Wikipedia:Bronchiole
 is_a: UBERON:0000117 ! respiratory tube
-relationship: branching_part_of UBERON:0002185 ! bronchus
+relationship: branching_part_of UBERON:0007196 ! tracheobronchial tree
 relationship: part_of UBERON:0002048 {source="MA"} ! lung
 relationship: part_of UBERON:0007196 {source="FMA"} ! tracheobronchial tree
 relationship: part_of UBERON:0010368 {source="ncit"} ! pulmonary lobule
```
Comment (anitacaron) [inline src/ontology/uberon-edit.obo:44039]: `branching part of` and `part of` isn't redundant for this term?

Comment (aleixpuigb) [reply src/ontology/uberon-edit.obo:44039]: You are right, I will remove the relationship

Comment (bvarner-ebi) [reply src/ontology/uberon-edit.obo:44039]: the `part of` axiom was not inferred after I deleted it.
Could benefit from @dosumis's feedback...

**Comment by anitacaron** [comment]
Is the description of this issue help here? https://github.com/obophenotype/uberon/issues/2212

**Comment by aleixpuigb** [comment]
@bvarner-ebi, if everything looks fine for you, I will merge the PR

**Review by bvarner-ebi — APPROVED** [review]

**Pull request merged**